aws/logs_monitoring/lambda_function.py

```diff
@@ -607,10 +629,12 @@ def split(events):
         elif trace and DD_FORWARD_TRACES:
             traces.append(trace)
         else:
+            add_metadata_to_lambda_log(event)
```

Comment:
I would leave `split` to ONLY split events, and fix the issue in `awslogs_handler`. See my other comment.